pkg/server/leases/controller.go

```diff
@@ -0,0 +1,49 @@
+package leases
```

Comment:
nit: missing license header on all new files

Reply:
Should be done now. Let me know if I missed any.